src/todo.js

```diff
@@ -0,0 +1,142 @@
+let tasks = [{
+  description: 'Task 1',
+  completed: false,
+  index: 1,
+},
+{
+  description: 'Task 2',
+  completed: true,
+  index: 2,
+},
+// Add more tasks with different index values
+];
+
+function updateIndexes() {
+  tasks.forEach((task, index) => {
+    task.index = index + 1;
+  });
+}
+
+export function deleteTask(todoItem) {
+  todoItem.remove();
+}
+
```

Comment:
- [ ] Good job implementing `deleteTask` function, however, when we are deleting `item`, we are not updating our local storage, then when you refresh the page we can be able to see deleted items. In your case, only when the item is completed, the local storage used to be updated but let's do it for both, Kindly, fix it 💪.

![Screenshot 2023-07-20 112043](https://github.com/Hsuwaddyhtat22/To-Do-List-mini-PJ/assets/57408419/2c16b822-d9a7-4968-befb-209928c7bce8)